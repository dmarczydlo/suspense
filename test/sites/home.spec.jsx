import React from 'react';
import Home from '../../src/sites/home';

describe('home site', () => {
    let wrapper = null;

    it('should be rendered', () => {
        wrapper = global.shallow(<Home />);
        expect(wrapper).toMatchSnapshot();
    });
});
