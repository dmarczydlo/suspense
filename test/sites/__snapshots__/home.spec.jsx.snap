// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home site should be rendered 1`] = `
<div
  className="test"
>
  <Link
    name="child1"
    replace={false}
    to="/child/1"
  >
    child 1
  </Link>
  <Link
    name="child2"
    replace={false}
    to="/child/2"
  >
    child 2
  </Link>
</div>
`;
